pegasus/sites.v3/hourofcode.com/public/thanks.md.partial

```diff
@@ -0,0 +1,39 @@
+---
+title: Thanks for signing up to host an Hour of Code!
+layout: wide
+nav: how_to_nav
+
+social:
+  "twitter:card": player
+  "twitter:site": "@codeorg"
+---
+
+# Thanks for signing up to host an Hour of Code!
+
+<br>
+**The Hour of Code runs during {{ campaign_date/full }} and we'll be in touch about new tutorials and other exciting updates as they come out. In the meantime, what can you do now?**
+
+As a special thank you to Hour of Code organizers, we have a new poster set featuring Malala, Stephen Curry, Shakira and more available to order at a [50% discount from Amazon](https://www.amazon.com/promocode/A3QAYNZUZTSSNQ). This year, each set comes with 6 posters and 126 "I did the Hour of Code" stickers. With the discount you'll get them for less than our cost to make them. Supplies are limited, so order your posters soon. If you're not in the United States, you can [download and print all posters](https://hourofcode.com/promote/resources#posters).
```

Comment:
Should these two URLs also be extracted?

Reply:
Good question!  We will want to extract (https://hourofcode.com/promote/resources#posters) but keep the amazon url (since it's an external link).

Reply:
@breville All done! Please review.